README.md

```diff
@@ -2,6 +2,12 @@
 
 KimonoRSS is a tool for easily consuming JSON APIs from [KimonoLabs](https://www.kimonolabs.com) either in a ruby environment or as an RSS feed of entry history.
 
+## Developing
+
+You will need Postgres 9.3 or higher.  [Get Postgres.app](http://postgresapp.com/) and then use [the guide](http://postgresapp.com/documentation/cli-tools.html) to get set up.
+
```

Comment:
Totally optional, but I believe you can also add a line for installing via homebrew (if the system is a mac).  Homebrew++